resources/dicts/thoughts/alive/general.json

```diff
@@ -51,87 +49,1497 @@
+            "Is cursing a squirrel that just barely got away",
+            "Is wishing {PRONOUN/m_c/subject} had wings so {PRONOUN/m_c/subject} could catch birds easier"
+        ],
+        "main_status_constraint": [
+            "apprentice",
+            "warrior",
+            "deputy",
+            "leader"
+        ],
+        "main_living_status": [
+            "living"
+        ]
+    },
+    {
+        "id": "working_cat_warrior",
+        "thoughts": [
+            "Is hoping the deputy won't assign {PRONOUN/m_c/object} to the dawn patrol tomorrow",
+            "Is hoping the deputy assigns {PRONOUN/m_c/object} to the next patrol",
+            "Is wondering if {PRONOUN/m_c/subject} could handle r_c's position",
+            "Is complaining about being sent on dawn patrol",
+            "Is complaining about being sent on dusk patrol",
+            "Hopes to go to the next gathering",
+            "Is reporting something to r_c",
+            "Is worried r_c is overworking {PRONOUN/r_c/self}",
+            "Is hoping the deputy won't assign {PRONOUN/m_c/object} to the next patrol"
+        ],
+        "main_status_constraint": [
+            "apprentice",
+            "warrior"
+        ],
+        "random_status_constraint": [
+            "deputy",
+            "leader"
+        ]
+    },
+    {
+    "id": "adventurous_gen",
+    "thoughts": [
+        "Wants to explore far beyond the borders of {PRONOUN/m_c/poss} territory...",
+        "Wonders what lays beyond the distant horizon...",
+        "Is dreaming of one day discovering something new and exciting!",
+        "Is itching to run, run, run!",
+        "Wants to climb the tallest tree in the territory!",
+        "Is itching for some excitement around camp",
+        "Is considering bending the rules a bit... just this once",
+        "Is showing off a trinket {PRONOUN/m_c/subject} found while exploring",
+        "Is daydreaming about how much there is to see out in the world!",
+        "Is itching to explore the land beyond {PRONOUN/m_c/poss} Clan's territory",
+        "Wants to go on a journey",
+        "Wishes {PRONOUN/m_c/poss} leader would choose {PRONOUN/m_c/object} to go on a quest",
+        "Disappeared all day to explore the territory",
+        "Promises {PRONOUN/m_c/poss} Clanmates {PRONOUN/m_c/subject} won't do anything dangerous while out"
+    ],
+    "main_trait_constraint": [
+        "adventurous"
+        ]
+    },
+    {
+        "id": "adventurous_gen_to_alive_kits",
+        "thoughts": [
+            "Involves the kits in a game of make believe about traveling through a strange land",
+            "Encourages the kits to explore as much as possible"
+        ],
+        "random_status_constraint": [
+            "kitten"
+        ],
+        "main_trait_constraint": [
+            "adventurous"
+        ],
+        "random_living_status": [
+            "living"
+        ]
+    },
+    {
+        "id": "ambitious_gen",
```

Comment:
some of these thoughts should be separated into a new thought group that requires the leader as the random cat